packages/amplify-frontend-javascript/lib/frontend-config-creator.js

```diff
@@ -60,11 +60,11 @@ function createAWSExports(context, amplifyResources) {
       default: break;
     }
   });
-  generateAWSExportsFile(context, configOutput);
+  await generateAWSExportsFile(context, configOutput);
   return context;
 }
 
-function generateAWSExportsFile(context, configOutput) {
+async function generateAWSExportsFile(context, configOutput) {
```

Comment:
just a note. this async may not be necessary, but it is more explicit that this is returning a promise.
Got a bit lost in the trail of function returning promises at the start

Reply:
you can put an `await' in front of the `amplify.copyBatch` in stead of `return`.